test-app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,12 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
 
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.TTS_SERVICE" />
+        </intent>
+    </queries>
+
```

Comment:
Would you mind mentioning that this is needed in `docs/guides/tts.md`? I'm afraid we'll forget about it.